pkg/config/config.go

```diff
@@ -1273,6 +1273,15 @@ func (c *RuntimeConfig) ValidateRuntimes() error {
 
 func (c *RuntimeConfig) initializeRuntimeFeatures() {
 	for name, handler := range c.Runtimes {
+		versionOutput, err := cmdrunner.CombinedOutput(handler.RuntimePath, "--version")
+		if err != nil {
+			logrus.Errorf("Unable to determine version of runtime %s: %v", name, err)
```

Comment:
For consistency... :smile: 

```suggestion
			logrus.Errorf("Unable to determine version of runtime handler %q: %v", name, err)
```